web/app/store/Events.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015 Anton Tananaev (anton.tananaev@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.store.Events', {
+    extend: 'Ext.data.Store',
+    model: 'Traccar.model.Event',
+
+    proxy: {
+        type: 'rest',
+        url: '/api/events'
+    }
+});
```

Comment:
New line is missing. By the way, there is style and issues check for JavaScript code as well. Use `jshint` and `jscs`.